examples/02_investigating_dirty_categories.py

```diff
@@ -147,29 +150,98 @@
 
 # encoding simply a subset of the observations
 n_obs = 20
-employee_position_titles = values['employee_position_title'].head(
-    n_obs).to_frame()
+employee_position_titles = values["employee_position_title"].head(n_obs).to_frame()
 categorical_encoder = OneHotEncoder(sparse=False)
 one_hot_encoded = categorical_encoder.fit_transform(employee_position_titles)
 f3, ax3 = plt.subplots(figsize=(6, 6))
 ax3.matshow(one_hot_encoded)
-ax3.set_title('Employee Position Title values, one-hot encoded')
-ax3.axis('off')
+ax3.set_title("Employee Position Title values, one-hot encoded")
+ax3.axis("off")
 f3.tight_layout()
 
 #########################################################################
 # The corresponding is very sparse
 #
-# SimilarityEncoder can be used to replace one-hot encoding capturing the
-# similarities:
+# :class:`SimilarityEncoder` can be used to replace one-hot encoding
+# capturing the similarities:
 
 f4, ax4 = plt.subplots(figsize=(6, 6))
 similarity_encoded = similarity_encoder.fit_transform(employee_position_titles)
 ax4.matshow(similarity_encoded)
-ax4.set_title('Employee Position Title values, similarity encoded')
-ax4.axis('off')
+ax4.set_title("Employee Position Title values, similarity encoded")
+ax4.axis("off")
 f4.tight_layout()
 
 #########################################################################
 # Other examples in the dirty_cat documentation show how
 # similarity encoding impacts prediction performance.
```

Comment:
```suggestion
# [Other examples](https://dirty-cat.github.io/stable/auto_examples/) in the 
# dirty_cat documentation show how similarity encoding impacts prediction performance.
```
Better yet: actually mention which notebooks (but this would be hard to update).